servers/text_server.h

```diff
@@ -63,12 +66,12 @@ class TextServer : public Object {
 		JUSTIFICATION_CONSTRAIN_ELLIPSIS = 1 << 4,
 	};
 
-	enum LineBreakFlag {
+	enum LineBreakFlag { // LineBreakFlag can be passed in the same value as the JustificationFlag, do not use the same values.
```

Comment:
Isn't there a conflict between `JUSTIFICATION_NONE` and `BREAK_NONE` though? Or it doesn't matter since that's the default anyway?

Reply:
It's default (0), so it does not matter.